python/cuml/cuml/tests/test_device_selection.py

```diff
@@ -1011,3 +1012,23 @@ def test_dbscan_methods(train_device, infer_device):
     assert_dbscan_equal(
         ref_output, output, X_train_blob, model.core_sample_indices_, eps
     )
+
+
+@pytest.mark.parametrize("train_device", ["cpu", "gpu"])
+@pytest.mark.parametrize("infer_device", ["cpu", "gpu"])
+def test_random_forest_regressor(train_device, infer_device):
+    model = RandomForestRegressor()
+    with using_device_type(train_device):
+        model.fit(X_train_reg, y_train_reg)
+    with using_device_type(infer_device):
+        _ = model.predict(X_test_reg)
```

Comment:
Any reason we're not comparing the result to sklearn here? I don't think we absolutely _have to_, but we should probably add a comment explaining why we're not.